### ISSUE TITLE
Use golang.org/x/mod for parsing Fyne version in go.mod

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/tools#5]
Use https://pkg.go.dev/golang.org/x/mod here:
https://github.com/fyne-io/tools/blob/9efbb19e1ba665912def685a290640d5c7822942/cmd/fyne/internal/commands/build.go#L173

**Comment by andydotxyz** [comment]
Make sure that whatever library we use it supports the new point release notation for Go since 1.21.0

**Comment by Jacalz** [comment]
I think `x/mod` is what the compiler uses internally for module parsing. I will be more or less the same as using the Go binary for it.

**Comment by andydotxyz** [comment]
Yes, but as noted in the issues in golang GitHub the compiler not matching the toolchain/library is the source of problems elsewhere so it felt important to note.

**Comment by Jacalz** [comment]
Just for the sake of clarity when someone looks at this in the future: I think the `go/version` package in the standard library might be doing the same thing.
